symplyphysics/laws/thermodynamics/relative_humidity.py

```diff
@@ -0,0 +1,46 @@
+from sympy import (Eq, solve, S)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless,
+    convert_to
+)
+
+# Description
+## Relative humidity is the ratio of water vapor pressure p to saturated vapor pressure p0 at a given temperature.
+
+## Law is: ph = (p / p_sat) * 100, where
+## ph - relative humidity,
+## p - water vapor pressure,
```

Comment:
Explain it better. What is water vapor? What is its pressure?

Reply:
Still not clear, what pressure do you mean here.
Actually I've tried to find, what pressure is meant here, and there is no clear answer to this. I think it should be the same for both saturated and actual vapor, so it can be any. But the easiest to find is static pressure, so it seems it is being used for calculations.
You can mention, that pressures on both vapors should be measured the same way, at the same temperature.